tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRConfig.java

```diff
@@ -359,29 +358,6 @@ public final Comparator<String> getComparatorRoot() {
         return (Comparator) (getCollatorRoot());
     }
 
-    private static final class CollatorHelper {
-        static final Collator EMOJI_COLLATOR = makeEmojiCollator();
-
-        private static final Collator makeEmojiCollator() {
-            final RuleBasedCollator col =
-                    (RuleBasedCollator)
-                            Collator.getInstance(ULocale.forLanguageTag("en-u-co-emoji"));
-            col.setStrength(Collator.IDENTICAL);
-            col.setNumericCollation(true);
-            col.freeze();
-            return col;
-        }
-
-        static final Collator ROOT_NUMERIC = makeRootNumeric();
-
-        private static final Collator makeRootNumeric() {
-            RuleBasedCollator _ROOT_COL = (RuleBasedCollator) Collator.getInstance(ULocale.ENGLISH);
-            _ROOT_COL.setNumericCollation(true);
-            _ROOT_COL.freeze();
-            return _ROOT_COL;
-        }
-    }
-
     public Collator getCollator() {
         return CollatorHelper.EMOJI_COLLATOR;
     }
```

Comment:
This CLDRConfig.getCollator has many callers -- might as well replace it everywhere with CollatorHelper.EMOJI_COLLATOR? There doesn't seem to be any advantage to indirect access through this method in CLDRConfig